Stripe/StripeiOSTests/FormSpecProviderTest.swift

```diff
@@ -574,4 +574,92 @@ class FormSpecProviderTest: XCTestCase {
         XCTAssertEqual(redirectToURL.urlPath, "next_action[redirect_to_url][url]")
         XCTAssertEqual(redirectToURL.returnUrlPath, "next_action[redirect_to_url][return_url]")
     }
+
+    func testUnparsableSpecIsSkipped() throws {
+        let e = expectation(description: "Loads form specs file")
+        let sut = FormSpecProvider()
```

Comment:
What does `sut` stand for? Can we use a more descriptive name?

Reply:
Looking above, seems like this is common in the rest of the tests, but it's still not a term of art I've ever heard before

Reply:
Means [System Under Test](https://en.wikipedia.org/wiki/System_under_test), it is a common term.